src/styles/getModalStyles.js

```diff
@@ -96,6 +96,24 @@ export default (type, windowDimensions) => {
             animationIn = 'fadeInLeft';
             animationOut = 'fadeOutLeft';
             break;
+        case CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED:
+            modalStyle = {
+                margin: 0,
+                flexDirection: 'row-reverse',
+                justifyContent: 'end',
+            };
+            modalContainerStyle = {
+                width: isSmallScreen ? '100%' : '40%',
```

Comment:
cc @shawnborton we might want to use a constant for the right side pane instead of the `40%`? I know there wasn't really a clear guideline for this part.

Reply:
Yeah maybe a `max-width`? `450px` maybe?

Reply:
What if we just reuse the variable for the LHN? So basically at mobile breakpoints, this should take up the whole screen (like the LHN) but at wider desktops, this should share the same LHN variable which would make it 375px. This is what we've been doing in all of our mockups.

Reply:
@yuwenmemon I think you'll need to pull master, and then you should find `sideBarWidth: 375` inside of `variables.js`